src/shared/components/palette/complex/complex.module.ts

```diff
@@ -38,17 +39,19 @@ import { WriteComplexFieldComponent } from './write-complex-field.component';
     ReadComplexFieldRawComponent,
     ReadComplexFieldTableComponent,
     ReadComplexFieldCollectionTableComponent,
+    CcdCYAPageLabelFilterPipe,
   ],
   entryComponents: [
     ReadComplexFieldComponent,
     WriteComplexFieldComponent,
   ],
-  exports: [
-    CcdPageFieldsPipe,
-    FieldsFilterPipe,
-    CcdTabFieldsPipe,
-    ReadFieldsFilterPipe,
-    WriteComplexFieldComponent
-  ]
+    exports: [
```

Comment:
_Nitpick:_
The indenting on this block is off - for some reason, it's using 4 spaces throughout, compared to 2 spaces for the rest of the file and 2 spaces as standard across the project.